src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -55,7 +55,7 @@ export class ServerConnectionIndicator extends LitElement {
         --circle-small-delay: 0ms;
 
         --circle-initial-opacity: 0.5;
-        --circle-initial-color: grayscale(0.5);
+        --circle-initial-color: grayscale(1);
```

Comment:
made the initial image completely grayscale

Reply:
Currently we're using the "disconnected" state in the empty view, as per that fix that was blocking linux. There was confusion around whether or not INITIAL was grayscale, remember? We determined that it was (I'm still confused as to why it's a state then) but this brings it more in line. I said I would make it look exactly like it did before so I guess I can revert this into a full fix instead.